src/navigator/TabNavigator.tsx

```diff
@@ -0,0 +1,73 @@
+import { createBottomTabNavigator } from '@react-navigation/bottom-tabs'
+import { NativeStackScreenProps } from '@react-navigation/native-stack'
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { StyleSheet } from 'react-native'
+import DAppsExplorerScreenSearchFilter from 'src/dappsExplorer/DAppsExplorerScreenSearchFilter'
+import WalletHome from 'src/home/WalletHome'
+import Discover from 'src/icons/navigator/Discover'
+import Home from 'src/icons/navigator/Home'
+import Wallet from 'src/icons/navigator/Wallet'
+import { Screens } from 'src/navigator/Screens'
+import { StackParamList } from 'src/navigator/types'
+import Colors from 'src/styles/colors'
+import { typeScale } from 'src/styles/fonts'
+import AssetsScreen from 'src/tokens/Assets'
+
+const Tab = createBottomTabNavigator()
+
+type Props = NativeStackScreenProps<StackParamList, Screens.TabNavigator>
+
+export default function TabNavigator({ route }: Props) {
+  const initialScreen = route.params?.initialScreen ?? Screens.WalletHome
+  const { t } = useTranslation()
+
+  return (
+    <Tab.Navigator
+      initialRouteName={initialScreen}
+      screenOptions={{
+        headerShown: false,
+        tabBarActiveTintColor: Colors.black,
+        tabBarInactiveTintColor: Colors.gray3,
+        tabBarLabelStyle: styles.label,
+      }}
+    >
+      <Tab.Screen
+        // TODO(act-1104) new assets screen
+        name={Screens.Assets}
+        // @ts-expect-error Type '{}' is missing the following properties from type 'Props': navigation, route
```

Comment:
can we avoid using `@ts-expect-error` here? I worry (in general) that will forget about suppressed errors and they eventually surface as errors to the user experience.

Reply:
This is just linking to the existing assets screen, but that screen be redone in act-1104 which should resolve this todo.